device/sycl/src/clusterization/clusterization_algorithm.sycl

```diff
@@ -153,15 +154,15 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
                         *mutex_ptr);
 
                     // Barrier used in the algorithm.
-                    traccc::sycl::barrier barry_r(item);
-                    const sycl::thread_id1 thread_id(item);
+                    const details::barrier barrier{item};
```

Comment:
RIP Barry.

Reply:
I did enjoy that name. But indeed, decided that we had enough fun with it. :thinking: